src/backend/web/validation.js

```diff
@@ -73,6 +73,13 @@ const queryValidationRules = [
     .isIn(['post', 'author'])
     .withMessage('invalid filter value')
     .bail(),
+
+  query('perPage', 'per_page needs to be empty or an integer').custom(
+    (value) => !value || Number.isInteger(+value)
+  ),
+  query('page', 'page needs to be empty or an integer').custom(
```

Comment:
This can be empty or a number between 1 and 1000 (i.e., 10 * 1000 = 10,000).

Reply:
page technically starts from 0, but that could be changed if you want